conductor/lib/maya_utils.py

```diff
@@ -103,6 +103,21 @@ def get_plugin_versions():
     return plugin_versions
 
 
+def save_current_maya_scene():
+    '''
+    Saves current Maya scene using standard save command
+    '''
+    cmds.SaveScene()
```

Comment:
What do you think about logging a debug statement here to give us breadcrumbs  e.g.` logger.debug("Saving maya scene..")`.

Reply:
Yep!   Added.